wire-ios-data-model/Source/ManagedObjectContext/NSManagedObjectContext+EncryptionAtRest.swift

```diff
@@ -126,12 +96,12 @@ extension NSManagedObjectContext {
             )
             saveOrRollback()
         } catch {
-            WireLogger.ear.error("failed to migrate existing data toward EAR: \(error)")
-            reset()
             throw error
         }
     }
 
+            WireLogger.ear.error("failed to migrate existing data toward EAR: \(error)")
+            reset()
```

Comment:
this probably shouldn't be here, I guess it should be in the `catch`

Reply:
Yeah somehow the merge got a bit messed up, already fixed